src/hooks/useDelayedDoubleClick.js

```diff
@@ -8,7 +8,7 @@ import useDebounce from './useDebounce'
  * Increasing the value lead to a perceivable delay between click and the placement of the marker. Decreasing the value
  * could lead to a double click never being captured, because of the limited time to have both click registered.
  */
-export const CLICK_TIMEOUT = 200
+export const CLICK_TIMEOUT = 300
```

Comment:
Why is this increase required? 200 ms Doesn't lead to a perceivable delay. 300 Does.

Reply:
Our users are not only people under 30. They need more time than 200 ms for double clicking.

Reply:
Tell that to the Leaflet maintainers: https://github.com/Leaflet/Leaflet/blob/fb0b2a169ebde77a8bed1715bb07d436b60328a2/src/dom/DomEvent.DoubleTap.js#L25